### DISCUSSION TITLE
Please add EditorConfig file

### DISCUSSION THREAD

**Discussion by spenserblack** [lowlighter/metrics#298] [Ideas]
I'm considering working on #253, but my editor is defaults to tab indentation. A [`.editorconfig`](https://editorconfig.org/) file can help make it easier to contribute to this repo.

(I'm not sure if discussions is the right place for this request, but I wasn't sure where else it should go :thinking:)

**Comment by lowlighter** [comment]
Indeed it can be problematic when editor is configured to format on save.
I'll add one to avoid large diffs in pull requests when reviewing 👍 

Otherwise, code is actually auto-formatted by GitHub actions when pushed on master to keep code styling consistent without annoying too much contributors by forcing the style in their PRs 🙂
